CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+## 🙏 Welcome
+
+Thank you for your interest in Openverse! We're so excited to bring new contributors into the fold in WordPress, Openverse and FOSS in general.
+
+## ❤️ Code of Conduct
+
+Please be aware that as a project of the WordPress Foundation, all Openverse repositories follow [the WordPress Etiquette](https://wordpress.org/about/etiquette/), a code of conduct that you should read and agree to abide by before contributing to WordPress projects.
+
+## 😍 Keeping in Touch
+
+For folks who'd like to be involved in our regular development conversations, please feel free to join us in the WordPress make chat (https://make.wordpress.org/chat/). You'll find us in the #openverse channel, of course! We have weekly community meetings on Tuesdays at 15:00 UTC (https://everytimezone.com/s/d1d42c7b). All are welcome! During the meetings we discuss ongoing work and assign new issues. Attendance is not mandatory to contribute, however. Any issues labeled as "good first issue" or "help wanted" in our repositories are all up for grabs. Just `@` a maintainer in the issue with questions or requesting the issue be assigned to you when you're ready to work on it. We also regularly discuss Openverse development issues outside the meeting times (like debugging problems we're having while working through an issue).
```

Comment:
`Just `@` a maintainer` can be difficult to understand for non-native new contributor. Something like just leave a comment mentioning a maintainer (with a list of maintainers or an example like `@obulat`?)